app/jobs/v3/services/update_broker_job.rb

```diff
@@ -65,7 +68,7 @@ def perform
           @warnings
         rescue => e
           begin
-            broker.update(state: previous_broker_state)
+            initial_broker.update(state: previous_broker_state)
```

Comment:
This PR fixes the issue, that the previous state could not be reset. Though I think it does not always make sense to let the broker return to the previous state.
For example when `@catalog_updater.refresh` fails to contact the broker to update the catalog and the previous_state was `AVAILABLE` the broker would return to state `AVAILABLE` even it is not reachable at the moment. I think in this case setting the brokers state to `SYNCHRONIZATION_FAILED` would be more appropriate. This still allows for broker.update request, but will let all service related operations fail fast.